draft-ietf-opsawg-pcap.md

```diff
@@ -55,8 +55,19 @@ The code to capture traffic, using low-level mechanisms in various
 operating systems, and to read and write network traces to a file was
 later put into a library named libpcap.
 
-This document describes the format used by tcpdump, and other
+This document describes the historical format used by tcpdump, and other
 programs using libpcap, to read and write network traces.
+This document describes version 2 of the pcap format.
+
+This document is published as historical, as there has existed for some time, an updated format originally called "pcapng", that replaces this file format.  See {{?I-D.ietf-opsawg-pcapng}}
```

Comment:
Right now, the plan seems to be the continue to call it "pcapng" and change what "ng" stands for, so should "originally" in "originally called 'pcapng'" be removed?

Reply:
Done.